core/backend/src/SheetIndex.ts

```diff
@@ -0,0 +1,326 @@
+import { BisCodeSpec, Code, CodeScopeProps, CodeSpec, ElementProps, EntityReferenceSet, IModelError, RelatedElementProps, SheetIndexEntryProps, SheetIndexFolderProps, SheetIndexReferenceProps, SheetReferenceProps } from "@itwin/core-common";
+import { InformationReferenceElement, Sheet } from "./Element";
+import { IModelDb } from "./IModelDb";
+import { Id64String, IModelStatus } from "@itwin/core-bentley";
+import { SheetIndexFolderOwnsEntries, SheetIndexOwnsEntries, SheetIndexReferenceRefersToSheetIndex, SheetReferenceRefersToSheet } from "./NavigationRelationship";
+
+/**
+ * A bis:InformationReferenceElement used to organize bis:Sheet instances into a hierarchy with the assistance
```

Comment:
Please link to the Sheet Index section of docs/bis/domains/drawings-sheets.md where people can learn what a sheet index is.

Reply:
Added a URL to the doc site.  I wasn't sure if there's a better way to do that or not.  If there is, I couldn't find an example of it in the repo.

Reply:
You can use `$docs` to reference the base url without hard coding a specific environment. Something like: `[SheetIndex]($docs/bis/domains/drawings-sheets#sheet-index)`

Reply:
Thanks for the suggestion.  Updated